manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -151,6 +151,7 @@ data:
   "trigger-csi-fullsync": "false"
   "pv-to-backingdiskobjectid-mapping": "false"
   "topology-aware-file-volume": "false"
+  "sanpshot-quota": "false"
```

Comment:
We may also address things that were missed out in milestone-1, let's call it `storage-quota-m2`

Reply:
ok will change it to "storage-quota-m2"